source/standalone/workflows/skrl/train.py

```diff
@@ -120,7 +120,8 @@ def main(env_cfg: ManagerBasedRLEnvCfg | DirectRLEnvCfg | DirectMARLEnvCfg, agen
     # set the environment seed
     # note: certain randomizations occur in the environment initialization so we set the seed here
     env_cfg.seed = args_cli.seed if args_cli.seed is not None else agent_cfg["seed"]
-
+    env_cfg.sim.device = args_cli.device
```

Comment:
Can you move it up to before `if args_cli.distributed`
and replace with `env_cfg.sim.device = args_cli.device if args_cli.device is not None else env_cfg.sim.device`

Reply:
Also need to remove L139-142 (multi-gpu training config). It is a duplication of L-110-111.